pkg/keyspace/keyspace.go

```diff
@@ -698,9 +695,9 @@ func (manager *Manager) PatrolKeyspaceAssignment() error {
 				currentStartID = keyspaces[0].GetId()
 				nextStartID = keyspaces[keyspaceNum-1].GetId() + 1
 			}
-			// If there are less than `keyspacePatrolBatchSize` keyspaces,
+			// If there are less than `maxTxnOps`,
```

Comment:
The original comment is correct
```suggestion
			// If there are less than `maxTxnOps` keyspaces,
```